### ISSUE TITLE
NoMethodError: undefined method `data'

### ISSUE THREAD

**Issue by lacostenycoder** [minimagick/minimagick#480]
I have a valid file path defined to a valid image file.

```
image = MiniMagick::Image.open(filepath)
image.data
NoMethodError: undefined method `data' for #<MiniMagick::Image:0x00007f837d8d1438>
from /Users/lacoste/.rvm/gems/ruby-2.3.6/gems/mini_magick-4.2.10/lib/mini_magick/image.rb:393:in `block in method_missing'
```

Here's what I'm running
Platform: MacOS  10.14.3
```
RubyGems Environment:
  - RUBYGEMS VERSION: 2.7.8
  - RUBY VERSION: 2.3.6 (2017-12-14 patchlevel 384) [x86_64-darwin18]
  - INSTALLATION DIRECTORY: /Users/lovebone/.rvm/gems/ruby-2.3.6
  - USER INSTALLATION DIRECTORY: /Users/lovebone/.gem/ruby/2.3.0
  - RUBY EXECUTABLE: /Users/lovebone/.rvm/rubies/ruby-2.3.6/bin/ruby
  - EXECUTABLE DIRECTORY: /Users/lovebone/.rvm/gems/ruby-2.3.6/bin
  - SPEC CACHE DIRECTORY: /Users/lovebone/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Users/lovebone/.rvm/rubies/ruby-2.3.6/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-18
  - GEM PATHS:
     - /Users/lovebone/.rvm/gems/ruby-2.3.6
     - /Users/lovebone/.rvm/gems/ruby-2.3.6@global
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
     - :sources => ["https://rubygems.org/"]
     - "benchmark" => false
     - "gem" => "--no-rdoc --no-ri --document=yri"
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /Users/lovebone/.rvm/gems/ruby-2.3.6/bin
     - /Users/lovebone/.rvm/gems/ruby-2.3.6@global/bin
     - /Users/lovebone/.rvm/rubies/ruby-2.3.6/bin
     - /Users/lovebone/.rvm/bin
     - /usr/local/opt/elasticsearch@5.6/bin
     - /Users/lovebone/.pyenv/shims
     - /Users/lovebone/.pyenv/bin
     - /usr/local/opt/mongodb@3.0/bin
     - /usr/local/opt/python/libexec/bin
     - /Users/lovebone/.nvm/versions/node/v10.15.3/bin
     - /Users/lovebone/.cargo/bin
     - /Users/lovebone/.composer/vendor/schickling/git-s3/bin
     - /usr/local/heroku/bin
     - /usr/local/bin
     - /usr/local/sbin
     - /usr/local/share/npm/bin
     - /usr/local/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
     - /usr/local/MacGPG2/bin
     - /opt/X11/bin
     - /usr/local/opt/go/libexec/bin
```

**Comment by janko** [comment]
You need to update to a newer version of the MiniMagick gem.

**Comment by lacostenycoder** [comment]
@janko what is version I need for ruby 2.3 ?

**Comment by janko** [comment]
You can just use the latest available, it should work with ruby 2.3.
